src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHetUnitTest.java

```diff
@@ -1,253 +1,311 @@
 package org.broadinstitute.hellbender.tools.walkers.annotator;
 
-import htsjdk.variant.variantcontext.*;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypeBuilder;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
 import org.broadinstitute.hellbender.GATKBaseTest;
 import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
 import org.broadinstitute.hellbender.utils.variant.GATKVCFHeaderLines;
 import org.testng.Assert;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.IntStream;
 
-import static org.mockito.ArgumentMatchers.refEq;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 public final class ExcessHetUnitTest extends GATKBaseTest {
-    private static double DELTA_PRECISION = .001;
-    private Allele Aref= Allele.create("A", true);
-    private Allele T = Allele.create("T");
-    private Allele C = Allele.create("C");
-    private int[] hetPLs = {240, 0, 240};
-    private int[] homRefPLs= {0, 60, 600};
+    private static final double DELTA_PRECISION = 1E-6;
+    private static final double DELTA_PRECISION_PARSE = 1E-3;
+    private static final Allele A_REF = Allele.create("A", true);
+    private static final Allele T = Allele.create("T");
+    private static final Allele C = Allele.create("C");
+    private static final int[] HET_PL_ARRAY = {240, 0, 240};
+    private static final int[] HOM_REF_PL_ARRAY = {0, 60, 600};
 
     @Override
     public String getToolTestDataDir() {
         return toolsTestDir + "walkers/annotator/";
     }
 
 
-    private Genotype makeG(String sample, Allele a1, Allele a2, int... pls) {
+    private static Genotype makeG(final String sample, final Allele a1, final Allele a2, final int... pls) {
```

Comment:
These things are like dandelions -- you take one out and another two pop up.  This is duplicated code that's also in VariantContextTestUtils (and a bunch of other places), but if you feel like you've already done your boy scout duty you can leave it.